common/thorhelper/thorsoapcall.cpp

```diff
@@ -1951,10 +1984,17 @@ class CWSCAsyncFor : implements IWSCAsyncFor, public CInterface, public CAsyncFo
     {
         if (soapTraceLevel > 6 || master->logXML)
         {
-            if (!contentEncoded)
-                master->logctx.mCTXLOG("%s: request(%s)", master->wscCallTypeText(), request.str());
+            StringBuffer contentStr;
+            if (contentEncoded)
+                contentStr.append(", content encoded.");
+            if ( (master->logXML) && (soapSepString.length() > 0) )
```

Comment:
check: I think the master->logXML test can be deleted.

Reply:
I left it and added comment - so we only do translation if LOG set in SOAPCALL args